src/System.Private.CoreLib/shared/System/ReadOnlySpan.Fast.cs

```diff
@@ -154,6 +154,24 @@ internal ReadOnlySpan(ref T ptr, int length)
 #endif
         }
 
+        public ref readonly T this[Index index]
```

Comment:
Same here/elsewhere. Public APIs need documentation.

Reply:
I'll add the documentation later. thanks @ahsonkhan